spanner/spec/spec_helper.rb

```diff
@@ -127,4 +128,110 @@ def create_test_database database_id, statements: []
     job.wait_until_done!
     database = job.results
   end
+
+    # Creates a temporary database with random ID (will be dropped after test)
+  # (re-uses create_database to create database with Albums/Singers schema)
```

Comment:
There's a leading space in the first line. The 1st comment-line doesn't quite align with the 2nd comment-line.